### ISSUE TITLE
idna version

### ISSUE THREAD

**Issue by avilella** [ewels/MultiQC#884]
**Description of bug:**

**MultiQC Error log:**
```python
Traceback (most recent call last):
  File "/usr/local/bin/multiqc", line 767, in <module>
    multiqc()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/bin/multiqc", line 413, in multiqc
    template_mod = config.avail_templates[config.template].load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2258, in load
    self.require(env, installer)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2278, in require
    items = working_set.resolve(reqs, env, installer)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 767, in resolve
    raise VersionConflict(tmpl % args)
pkg_resources.VersionConflict: idna 2.8 is installed but idna<2.8,>=2.5 is required by ['requests']
```

**MultiQC run details:**
```
[WARNING]         multiqc : MultiQC Version v1.7 now available!
[INFO   ]         multiqc : This is MultiQC v1.6.dev0
[INFO   ]         multiqc : Template    : default
[INFO   ]         multiqc : Searching '/home/dnanexus/CEGX_Run521'
```

- Operating System:  Ubuntu 14.04

**Comment by avilella** [comment]
Same issue after updating to head master:

```
[INFO   ]         multiqc : This is MultiQC v1.8.dev0
[INFO   ]         multiqc : Template    : default
[INFO   ]         multiqc : Searching '/home/dnanexus/CEGX_Run521'
Traceback (most recent call last):
  File "/usr/local/bin/multiqc", line 766, in <module>
    multiqc()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/bin/multiqc", line 411, in multiqc
    template_mod = config.avail_templates[config.template].load()
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2345, in load
    self.require(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2368, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/usr/local/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 789, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (idna 2.8 (/usr/share/dnanexus/lib/python2.7/site-packages), Requirement.parse('idna<2.8,>=2.5'), set(['requests']))
```

**Comment by ewels** [comment]
Reading the traceback shows that your installation of `idna` conflicts with the requirements of the `requests` package. You have `v2.8` installed and requests requires `<2.8,>=2.5`.

You should be able to fix this by manually reinstalling the correct version of idna:
```
pip install --upgrade --force-reinstall idna
```

**Comment by avilella** [comment]
I added this line before the pip install for MultiQC, but it did not solved the issue for Ubuntu 14.04. I since then re-built for Ubuntu 16.04, so the issue is resolved on my end. Thanks.
